lib/crypto.js

```diff
@@ -2,23 +2,29 @@
 
 var crypto = require('crypto');
 
-function encrypt(key, text){
-  var IV = new Buffer(crypto.randomBytes(16));
-  var cipher = crypto.createCipheriv('aes-256-ctr', key, IV)
-  var crypted = cipher.update(text,'utf8','hex')
+function encrypt(key, text) {
+  const hashedAccessKey = crypto.createHash('sha256').update(key).digest('hex');
+  const keyBuffer = Buffer.from(hashedAccessKey, 'hex');
+  const iv = new Buffer(crypto.randomBytes(16));
+  var cipher = crypto.createCipheriv('aes-256-ctr', keyBuffer, iv)
+  var crypted = cipher.update(text, 'utf8', 'hex')
   crypted += cipher.final('hex');
-  return crypted;
+  return `${iv.toString('hex')}:${crypted.toString()}`;
```

Comment:
Template strings also is ES6 stuff... might break on some old dude browser :laughing: